common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoScorer.java

```diff
@@ -96,10 +94,10 @@ public ScoreResponse score(ScoreRequest request) {
       ShapleyType requestedShapleyType = shapleyType(request.getRequestShapleyValueType());
       switch (requestedShapleyType) {
         case TRANSFORMED:
```

Comment:
qq - was there an option for `BOTH` ever or was it decided that giving both results was too much?

Reply:
I don't think mojo2 supports both the transformed and original at the same time yet

Reply:
ref https://github.com/h2oai/mojo2/blob/643128c00f511ca7c9c38abdf4598f3edc373541/java/mojo2-cli/src/main/java/ai/h2o/mojos/ExecuteMojo.java#L142